draft-pauly-ohai-svcb-config.md

```diff
@@ -247,6 +305,17 @@ different clients. Clients SHOULD employ some technique to mitigate key
 targeting attack. Oblivious gateways that are detected to use targeted
 key configurations per-client MUST NOT be used.
 
+Oblivious gateways that are accessed based on SVCB discovery are required
+to offer a well-known oblivious gateway request path. This is done in
+part to make client-targeting attacks more difficult. If the gateway request
+path is communicated in the SVCB parameters, individual clients could
+receive unique paths which could be used to identify them upon making
+requests via a relay. While it is still possible for different clients to
+receive different gateway names, this requires provisioning more gateway
+names (which has overhead in DNS configuration and TLS certificate
+generation), and allows relays to more easily enforce allow-lists of
+known gateway names without needing to also check request paths.
```

Comment:
The above attack of different gateway names is possible with an malicious target aiming to identify clients. The attack can be mitigated by the client by using an pre-configured encrypted resolver to retrieve the SVCB records. The client can then compare this against the records retrieved using DNR/DDR to identify any targeted attack.

Reply:
This is changed now to not use a well-known

Reply:
This is changed now to not use a well-known